### DISCUSSION TITLE
Custom lable text

### DISCUSSION THREAD

**Discussion by rezafaraji93** [patrykandpatrick/vico#376] [Questions] [answered]
How can we implement and show custom label text. for example I want to have three lines in my label text component on my line chart with two chart entries, first line for x value, second and third the y value of the line charts. Thanks for your help.

**Comment by patrickmichalik** [comment, marked as answer]
Hello! You can create a custom `MarkerLabelFormatter` to achieve this. Please see the `labelFormatter` field of `MarkerComponent`.
